mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -412,6 +415,18 @@ public void onWidgetUpdated(OpenHABWidget widget) {
         }
     }
 
+    @Override
+    public void onLoadFailure(int statusCode, Headers headers, byte[] responseBody, Throwable e) {
+        int errorMsgRes = R.string.error_sitemap_generic_load_error;
```

Comment:
I wonder whether it wouldn't make more sense to use the same HTTP-status-code-to-error-message mapping as in OpenHABMainActivity instead of putting the exception message in.